core/paddlefl_mpc/operators/conv_op.h

```diff
@@ -142,35 +142,53 @@ inline bool IsExpand(const std::vector<int64_t>& filter_dim,
 template <typename DeviceContext, typename T>
 inline void ResizeToChannelFirst(const framework::ExecutionContext& context,
                                  const Tensor* input,
-                                 Tensor* transformed_input) {
+                                 Tensor* transformed_input,
+                                 bool is_output = false) {
   // extra 1 for leading share dim S
   int dim = input->dims().size() - 2 - 1;
   if (dim == 3) {
     // input
     transformed_input->Resize(input->dims());
 
-    // SNDHWC -> NCSDHW
     auto in_dims_vec = framework::vectorize(input->dims());
-    in_dims_vec[0] = input->dims()[1];
-    in_dims_vec[1] = input->dims()[5];
-    in_dims_vec[2] = input->dims()[0];
-    in_dims_vec[3] = input->dims()[2];
-    in_dims_vec[4] = input->dims()[3];
-    in_dims_vec[5] = input->dims()[4];
+    if (is_output) {
```

Comment:
please doc what does 'is_output' flag mean, and why the flag leads to following clause.